test-data/unit/check-typeguard.test

```diff
@@ -56,12 +56,13 @@ def main(a: object, b: object) -> None:
 
 [case testTypeGuardIsBool]
 from typing_extensions import TypeGuard
-def f(a: TypeGuard[int]) -> None: pass
```

Comment:
What about a function returning `List[TypeGuard[bool]]`?

Reply:
Good question! I will add this test case.